.circleci/config.yml

```diff
@@ -64,23 +41,24 @@ workflows:
           filters:
             tags:
               ignore: /.*/
+      - dps/helm_lint:
+          name: helm_lint
```

Comment:
why have you added the `name` here?  Couldn't you just add  the `requires` as `dps/helm_lint`?